pkg/minikube/assets/vm_assets.go

```diff
@@ -234,9 +240,9 @@ func (m *BinAsset) loadData(isTemplate bool) error {
 
 	m.length = len(contents)
 	m.reader = bytes.NewReader(contents)
-	glog.V(1).Infof("Created asset %s with %d bytes", m.AssetName, m.length)
+	glog.V(1).Infof("Created asset %s with %d bytes", m.SourcePath, m.length)
```

Comment:
nit: %q

Reply:
It's a filepath -- normally it isn't necessary to quote.